projects/plugins/jetpack/extensions/blocks/premium-content/edit.js

```diff
@@ -217,6 +218,44 @@ function Edit( props ) {
 
 	const { isSelected, className } = props;
 
+	useEffect( () => {
+		if ( isSelected ) {
+			return; // If this block is selected then leave the focused tab as it was.
+		}
+
+		const selectedBlock = select( 'core/block-editor' ).getSelectedBlock();
+		if ( ! selectedBlock ) {
+			return; // Sometimes there isn't a block selected, e.g. on page load.
+		}
+
+		// Confirm that the selected block is a descendant of this one.
+		if (
+			! select( 'core/block-editor' ).getBlockParents( selectedBlock.clientId ).includes( clientId )
+		) {
+			return;
+		}
+
+		if (
```

Comment:
The Ifs below *are* entered when expected if the premium-content block is selected and then selecting in the editing view, but not in the list view. Maybe that's doing:
 1. Deselect the premium content block
 2. Render
 3. Select the chosen block
 4. Render
???